covid_xprize/examples/prescriptors/neat/prescribe.py

```diff
@@ -155,41 +166,52 @@ def prescribe(start_date_str: str,
                 # Map prescription to integer outputs
                 prescribed_ips = (prescribed_ips * ip_max_values_arr).round()
 
-                # Add it to prescription dictionary
+                # Add it to prescription dictionary for the full ACTION_DURATION
                 country_name, region_name = geo.split('__')
                 if region_name == 'nan':
                     region_name = np.nan
```

Comment:
These five lines are indented below, so that they are run ACTION_DURATION times.